src/pages/office/components/additional-information/additional-information.ts

```diff
@@ -0,0 +1,33 @@
+import { Component, Input, Output, EventEmitter, OnChanges } from '@angular/core';
+import { FormGroup, FormControl } from '@angular/forms';
+
+@Component({
+  selector: 'additional-information',
+  templateUrl: 'additional-information.html',
+})
+export class AdditionalInformationComponent implements OnChanges {
+
+  @Input()
+  additionalInformation: string;
+
+  @Input()
+  formGroup: FormGroup;
+
+  @Output()
+  additionalInformationChange = new EventEmitter<string>();
```

Comment:
Did you have a go at passing a dispatcher down from the parent?

Reply:
Not yet, will have a try for this PR, see how it compares.

Reply:
As discussed, going with the `EventEmitter` for now.

Reply:
As discussed after comparison with #331, going with `EventEmitter` for now.